### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by rohankatyal29**
UI for difficulty buckets: user can add/delete words in difficulty buckets from the options page
Delete words in the difficulty buckets using the context menu 
Tooltip explaining the learnt words feature
Minor code refactoring

### CODE REVIEW

**Comment by ankit-m** [comment]
@rohankatyal29 you forgot to remove the `.RHistory` file

**Comment by rohankatyal29** [comment]
@ankit-m: Thanks for pointing that out, it's been removed now

File: lib/scripts/services/contextMenu.js [hunk 1]
```diff
@@ -124,30 +124,17 @@ export class ContextMenu {
       }
     }
 
-    if (isTranslatedWord) {
-
-      // Hash storage format:
-      // format: {targetLanguage: {word1: E/N/H, word2: E/N/H}}
-      // E -> easy, N -> normal, H -> hard
+    if (isTranslatedWord){
```
Comment (Ceilican) [inline lib/scripts/services/contextMenu.js]: space between ) and {

File: lib/scripts/services/contextMenu.js [hunk 2]
```diff
@@ -124,30 +124,17 @@ export class ContextMenu {
...
+        // Hash storage format:
+        // format: {word1: E/N/H, word2: E/N/H}
```
Comment (Ceilican) [inline lib/scripts/services/contextMenu.js]: Why did you decide to stop having separate buckets for different target languages?  (I am just curious about your reasons)

**Comment by Ceilican** [comment]
Thanks!

File: lib/scripts/services/contextMenu.js (hunk 1 above)
Comment (rohankatyal29) [inline lib/scripts/services/contextMenu.js]: Updated in the new PR

File: lib/scripts/services/contextMenu.js (hunk 2 above)
Comment (rohankatyal29) [inline lib/scripts/services/contextMenu.js]: @Ceilican: It wasn't providing me with any real benefit in the first place. The check for the target language was unnecessary

**Pull request merged**